### DISCUSSION TITLE
Password mask for password-confirmation field

### DISCUSSION THREAD

**Discussion by dinacuatro** [primefaces/community#2836] [PrimeVue]
I have two password fields, one for password and another for confirmation.
I would like to have only one mask/unmask icon, which will change both fields.

I tried to combine `toggleMask` with `unmasked` attribute, the problem is that looks like `toggleMask` does not fire any event. 
Is it like this? Any solution for that?

**Comment by j0rgedev** [comment]
Hey @dinacuatro. Unfortunately, it seems like the password component doesn't fire any event. You could try making a wrapper component

**Comment by launv** [comment]
Hi @dinacuatro . I think that you should use the template for the icon `https://primeng.org/password#api.password.templates`. Then you can emit an event when click eye button.
